generated/ruby/grpc-google-iam-v1/grpc-google-iam-v1.gemspec

```diff
@@ -0,0 +1,26 @@
+# -*- ruby -*-
+# encoding: utf-8
+
+Gem::Specification.new do |s|
+  s.name          = 'grpc-google-iam-v1'
+  s.version       = '0.6.8'
```

Comment:
Could you update the version to `0.1.0`?  Or `1.0.0` might be even okay, because `grpc` is already `1.0` and `googleapis-common-protos` package is already `1.3.1`.